modules/brushingandlinking/include/modules/brushingandlinking/brushingandlinkingmanager.h

```diff
@@ -231,6 +250,17 @@ class IVW_MODULE_BRUSHINGANDLINKING_API BrushingAndLinkingManager : public Seria
         std::function<void(BrushingAction, BrushingTarget, const BitSet&, std::string_view)>
             callback);
 
+    /**
+     * Returns the types of actions causing the owner (BrusingAndLinkingInport) to invalidate.
```

Comment:
affects both ports, suggesting "BrushingAndLinking port" instead.

Reply:
Right now it only applies to inports, but it might make sense to change it.